test/printer_test.rb

```diff
@@ -71,6 +72,17 @@ def setup
 
     print_stuff
   end
+  
+  should "return an array of jobs" do
+    stub_connection
+    fake_connection.expects(:post).with('/jobs', {:printerid => "printer"}).returns(real_jobs_hash)
+
+    print_job_array = get_all_jobs
+    assert print_job_array.is_a?(Array)
+    assert print_job_array.map(&:class).uniq.length.equal?(1)
+    assert print_job_array.map(&:class).uniq.first.equal?(CloudPrint::PrintJob)
+    assert print_job_array.length.equal?(3)
```

Comment:
@josephecombs I'd like to see this change made before merging. Also, there are merge conflicts now due to my changes (sorry).